webapp/src/app.tsx

```diff
@@ -1183,9 +1194,12 @@ export class ProjectView
             })
     }
 
-    clearLog() {
-        let logs = this.refs["logs"] as logview.LogView;
-        logs.clear();
+    clearSerial() {
+        this.serialEditor.clear()
+        let simLogs = this.refs["simLogs"] as logview.LogView
+        let devLogs = this.refs["devLogs"] as logview.LogView
```

Comment:
Is it guaranteed that those are defined?

Reply:
Fixed